### PR TITLE
Add telemetry for conflict markers

### PR DESCRIPTION
**Pull request by hediet**
Adresses #164884

### CODE REVIEW

**Review by lramos15 — CHANGES_REQUESTED** [review]
Believe some imports should go to `import type` for just a clearer distinction of where something is being used. 

These are just suggestions, I clicked request changes so it wouldn't auto-approve. If you disagree feel free to re-request me :)

File: extensions/merge-conflict/src/documentMergeConflict.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 import * as interfaces from './interfaces';
 import * as vscode from 'vscode';
+import TelemetryReporter from '@vscode/extension-telemetry';
```
[the remark this answers is not in the document]
Comment (hediet) [reply extensions/merge-conflict/src/documentMergeConflict.ts:7]: Btw. in github you can directly suggest:

```suggestion
import type TelemetryReporter from '@vscode/extension-telemetry';
```

File: extensions/merge-conflict/src/services.ts [hunk 2]
```diff
@@ -9,20 +9,25 @@ import CommandHandler from './commandHandler';
...
 
 const ConfigurationSectionName = 'merge-conflict';
 
 export default class ServiceWrapper implements vscode.Disposable {
 
 	private services: vscode.Disposable[] = [];
+	private telemetryReporter: TelemetryReporter;
 
 	constructor(private context: vscode.ExtensionContext) {
+		const { aiKey } = require('../package.json') as { aiKey: string };
```
[the remark this answers is not in the document]
Comment (hediet) [reply extensions/merge-conflict/src/services.ts:22]:
```suggestion
		const { aiKey } = context.extension.packageJSON as { aiKey: string };
```

File: extensions/merge-conflict/src/documentMergeConflict.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (lramos15) [reply extensions/merge-conflict/src/documentMergeConflict.ts:7]: I could never figure out the UI to do that, I think someone showed me at some point. I always click the blue plus icon and that brings up the comment widget

[the remark this answers is not in the document]
Comment (hediet) [reply extensions/merge-conflict/src/documentMergeConflict.ts]: image.png
image

**Review by lramos15 — APPROVED** [review]

**Pull request merged**